activerecord/lib/active_record/core.rb

```diff
@@ -609,6 +613,14 @@ def strict_loading?
       @strict_loading
     end
 
+    def _create_root_load_tree_node(siblings:)
```

Comment:
Do we want some sort of feature flag setting around this and the association one as well?